Objects/frameobject.c

```diff
@@ -1118,6 +1123,12 @@ PyFrame_GetBack(PyFrameObject *frame)
     return back;
 }
 
+PyObject*
+PyFrame_GetLocals(PyFrameObject *frame)
```

Comment:
Please document the new function:

* https://docs.python.org/dev/c-api/reflection.html#c.PyFrame_GetBack
* What's New in Python 3.11 > C API > New Features.

Reply:
I merged my https://github.com/python/cpython/pull/32051 PR